common-utils/poetry.lock

```diff
@@ -354,89 +354,19 @@ pycparser = "*"
 
 [[package]]
 name = "charset-normalizer"
-version = "3.1.0"
+version = "2.1.1"
```

Comment:
Just curious: why did charset-normalizer go to a lower version? I'm guessing these changes were triggered by a version change in pyproject.toml?

Reply:
oh, I installed `prefect-snowflake` to create the example and then removed it.  Looks like it downgraded to version that supported both `requests` and `prefect-snowflake`. Then left it when I removed the `prefect-snowflake` lib cause it was still within the version ranges.